### ISSUE TITLE
external SMB files shares no longer working

### ISSUE THREAD

**Issue by jfrancais** [linuxserver/docker-nextcloud#143]
[linuxserverurl]: https://linuxserver.io
[![linuxserver.io](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver_medium.png)][linuxserverurl]

If you are new to Docker or this application our issue tracker is **ONLY** used for reporting bugs or requesting features. Please use [our discord server](https://discord.gg/YWrKVTn) for general support.

------------------------------

## Expected Behavior
All my SMB external mounts no longer work.  They either timeout or return an internal server error.

## Current Behavior
Should be able to browse and access files on external file shares

## Steps to Reproduce

1. log on to server
2. click into external file share

## Environment
**OS:**      Unraid latest
**CPU architecture:** x86_64
**How docker service was installed:**      
unraid applications, updated via the unraid gui as standard

## Command used to create docker container (run/create/compose/screenshot)

unraid gui

## Docker logs

-------------------------------------
          _         ()
         | |  ___   _    __
         | | / __| | |  /  \ 
         | | \__ \ | | | () |
         |_| |___/ |_|  \__/

Brought to you by linuxserver.io
-------------------------------------

To support LSIO projects visit:
https://www.linuxserver.io/donate/
-------------------------------------
GID/UID
-------------------------------------

User uid:    99
User gid:    100
-------------------------------------

[cont-init.d] 10-adduser: exited 0.
[cont-init.d] 20-config: executing... 
[cont-init.d] 20-config: exited 0.
[cont-init.d] 30-keygen: executing... 
using keys found in /config/keys
[cont-init.d] 30-keygen: exited 0.
[cont-init.d] 40-config: executing... 
[cont-init.d] 40-config: exited 0.
[cont-init.d] 50-install: executing... 
[cont-init.d] 50-install: exited 0.
[cont-init.d] 60-memcache: executing... 
[cont-init.d] 60-memcache: exited 0.
[cont-init.d] 99-custom-files: executing... 
[custom-init] no custom files found exiting...
[cont-init.d] 99-custom-files: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.

**Comment by tobbenb** [comment]
It's the same here and it's an issue with nextcloud, so not something we can do anything about.
https://github.com/nextcloud/server/issues/20622

**Comment by wiswedel** [comment]
duplicate of nextcloud/server#20622

**Comment by tobbenb** [comment]
To add to the duplicate message from wiswedel, the fix will be in 18.0.5 released in early June. Go to the issue on nextclouds GitHub for more info.

**Comment by maximiliankaprolat** [comment]
I had the same issue. Currently there is a problem with the samba version which is fixed in version 19.x. 

The version linuxserver/nextcloud:18.0.4-ls78 works for me.
After downgrading to this version the SMB shares worked again.

**Comment by 0x4161726f6e** [comment]
I'm running into this problem and from what I have read elsewhere this might be a issue of using samba-client rather than smbclient.

**Comment by j0nnymoe** [comment]
SMB mounts working great here with 19.0.5.

**Comment by 0x4161726f6e** [comment]
Are you using user_external for SMB auth?

**Comment by j0nnymoe** [comment]
Yep - couple users on nextcloud using their own individual SMB logins

**Comment by 0x4161726f6e** [comment]
I guess I need to upgrade from 17.0.9 then.
